api/src/main/java/org/openmrs/module/fhir2/providers/r3/EncounterFhirResourceProvider.java

```diff
@@ -188,12 +188,15 @@ public IBundleProvider getEncountersWithMedicationRequestsSearch(
 		}
 		
 		if (status != null && !status.isEmpty() && status.getValue().equalsIgnoreCase("active")) {
-			// encounter must have a medication request that is neither completed nor cancelled (expired is okay, since we are using another definition of expired for dispensing purposes)
+			// encounter must have a medication request that is neither completed nor cancelled nor declined (expired is okay, since we are using another definition of expired for dispensing purposes)
 			HasOrListParam notCompletedHasParam = new HasOrListParam()
-			        .add(new HasParam("MedicationRequest", "encounter", "status:not", "completed"));
+			        .add(new HasParam("MedicationRequest", "encounter", "fulfillerStatus:not", "completed"));
+			HasOrListParam notDeclinedHasParam = new HasOrListParam()
+			        .add(new HasParam("MedicationRequest", "encounter", "fulfillerStatus:not", "declined"));
 			HasOrListParam notCancelledHasParam = new HasOrListParam()
 			        .add(new HasParam("MedicationRequest", "encounter", "status:not", "cancelled"));
-			params.setHasAndListParam(new HasAndListParam().addAnd(notCancelledHasParam).addAnd(notCompletedHasParam));
+			params.setHasAndListParam(
+			    new HasAndListParam().addAnd(notCancelledHasParam).addAnd(notDeclinedHasParam).addAnd(notCompletedHasParam));
```

Comment:
this query has been tweaked to include "declined" medication requests as ones that are excluded when determining "active"; this requires the new fulfiller status enum values defined in TRUNK-6176

Reply:
Defer to @ibacher and you @mogoodrich as to whether this "active" indication is something that should be changed like this, but seems fine to me

Reply:
This search endpoint has been designed as a custom endpoint for the Dispensing app, so it really needs to do whatever business logic the Dispense app requires... that being said, it's a good point that if that is the intent, the endpoint should be be commented and/or named better to make that clear... I will take a look next week.